packages/server/src/fhir/operations/groupexport.ts

```diff
@@ -30,7 +30,7 @@ export async function groupExportHandler(req: FhirRequest): Promise<FhirResponse
 
   // Start the exporter
   const exporter = new BulkExporter(ctx.repo, since, types);
-  const bulkDataExport = await exporter.start(concatUrls(baseUrl, 'fhir/R4' + req.pathname));
+  const bulkDataExport = await exporter.start(concatUrls(baseUrl, 'fhir/R4/' + req.pathname));
```

Comment:
```suggestion
  const bulkDataExport = await exporter.start(concatUrls(baseUrl, 'fhir/R4', req.pathname));
```
Does this work?

Reply:
@ThatOneBro  - yeah, i had the same thought...  Our current `concatUrls()` method does not support this.  we could though.  might be out of scope for this PR.